### DISCUSSION TITLE
How do we make clear to audience participants the relationship between GSF and the meetup group?

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/discussions#11] [Green Software Meetups] [answered]
We want to respect the people who are putting in all the effort to run meetups events related to green software, respect that we are simply supporting their hard efforts, they don't work for the GSF. At the same time in exchange for the sponsorship and our support for running meetup events we have some expectations on our side, adhering to our code of conduct, our content policy etc...

Perhaps we need to both make the relationship clear, we don't own the meetup groups but the meetup organizers at the same time are not official spokespeople for the GSF, they are independent thinkers.

Perhaps a message at the top of every meetup group page to signal our relationship: "Green Software XXX is an independently organized Green Software meetup sponsored by the Green Software Foundation"

**Comment by jawache** [comment, marked as answer]
The way our meetup groups are (will) function is via sponsorship and organizer list.

So if the GSF created the meetup group we are the main organizer and ultimate control over who the other organizers are sits with us, the group could never leave the fold of the GSF umbrella since we created it.

If someone else created the group and agreed to come under the GSF umbrella (we'd sponsor, pay their meetup fees) then we are not the main organizer of the meetup group and we have little influence in how it's run/managed.

We are currently 100% the first model, but a small handful of other meetup groups may want to join our umbrella group in the future.

Regardless of the model, the organizers of a meetup group should have a mandate and agency to run the group as they see fit, we should not micromanage anything here. We manage through setting policies and enforcing policies.

So in terms of language let's get clear.

**GSF Supported Meetup Groups.**
- For new meetup groups that we create and launch.
- The GSF account is listed as the main organizer of the meetup group.
- GSF decides who the other organizers are.
- GSF pays all the fees associated with that group.
- If the group/organizers are not meeting our code of conduct or any of our other policies, **we can remove/replace them**.
- Disclaimer at the top "Green Software XXX is an independently organized Green Software meetup **supported** by the Green Software Foundation"

**GSF Sponsored Meetup Groups.**
- For existing meetup groups that want to come under our umbrella.
- The GSF account is **not** an organizer of the meetup group.
- GSF cannot decide who the other organizers are.
- GSF pays all the fees associated with that group.
- If the group/organizers are not meeting our code of conduct or any of our other policies, **we can stop sponsoring them**.
- Disclaimer at the top "XXX is an independently organized Green Software meetup **sponsored** by the Green Software Foundation"

**Reply by jawache** [reply]
Agreed in Community WG call

**Comment by adamj89** [comment]
@jawache just reading through this. Totally agree as we have a couple of groups who would like to be sponsored. 

Something I found out in the investigation process is that for the Sponsored group type, we still have to be made a group organiser in order to invite them to meetup pro. So we might have to amend that line to clarify that we won't be an active organiser but that technically we need to have the privileges set in meetup.com.
